bin/handler-sensu.rb

```diff
@@ -131,7 +133,7 @@ def parse_remediations(remediations, occurrences, severity)
   # Issue a check via the API
   def trigger_remediation(check, subscribers)
     api_request(:POST, '/request') do |req|
-      req.body = JSON.dump('check' => check, 'subscribers' => subscribers)
+      req.body = JSON.dump('check' => check, 'subscribers' => [subscribers], 'creator' => 'SENSU', 'reason' => 'Try to fix')
```

Comment:
:+1: for more context. I wonder if maybe something more like `'creator' => 'sensu-plugins-sensu', 'reason' => 'Auto remediation triggered'` would make it clearer where it comes from and why it is doing it.

Reply:
Yes. It's better.